discovery/uyuni/uyuni.go

```diff
@@ -52,9 +52,10 @@ const (
 
 // DefaultSDConfig is the default Uyuni SD configuration.
 var DefaultSDConfig = SDConfig{
-	Entitlement:     "monitoring_entitled",
-	Separator:       ",",
-	RefreshInterval: model.Duration(1 * time.Minute),
+	Entitlement:      "monitoring_entitled",
+	Separator:        ",",
+	RefreshInterval:  model.Duration(1 * time.Minute),
```

Comment:
The same  constant can be  used here

Reply:
I don't think a constant is really needed here.

Reply:
I don't think too.
For consistency with the test, maybe choose `1 * time.Minute` or `60 * time.Second` but not both even if it doesn't really matter